lib/Data/Validator.pm

```diff
@@ -164,16 +165,28 @@ sub validate {
             $used++;
         }
         elsif(exists $rule->{default}) {
-            my $default = $rule->{default};
-            $args->{$name} = Mouse::Util::TypeConstraints::CodeRef($default)
-                ? $default->($self, $rule, $args)
-                : $default;
+            push @defaults, $rule;
```

Comment:
Is it followed in this case?
https://github.com/gfx/p5-Data-Validator/pull/8/files#diff-a637ac8d3e147affaa8858160e2ccbe3R134

Reply:
Could you give me a little more advise? :)

Do you mean necessity need for skipping exclusive parameters?